src/wallet/wallet.cpp

```diff
@@ -30,7 +30,6 @@ void Wallet::Stop() {
         scheduleTask_.join();
     }
     threadPool_.Stop();
-    spdlog::info("Wallet stopped.");
```

Comment:
I would suggest keeping these logs when shutting down. However, you may change it to debug or trace level